packages/studio-base/src/players/RosbridgePlayer.ts

```diff
@@ -425,6 +415,11 @@ export default class RosbridgePlayer implements Player {
     this.#emitState();
   }
 
+  public scaleBackArray(value: number, min: number, max: number): number {
```

Comment:
this isn't really taking in an array as the python version did. Could you rename to `scale_back_int16` or something similar?